.changeset/hungry-eggs-look.md

```diff
@@ -0,0 +1,6 @@
+---
+'@cypress-design/react-button': minor
+'@cypress-design/vue-button': minor
```

Comment:
I believe these are typically `patch` bumps

Reply:
I was thinking this too, but I'm fine either way I suppose.